portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -29,58 +29,46 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Date;
-
-
- 
+import com.google.gson.Gson; 
 
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
 	private List<Comment> comments;
 
- 	@Override
-    public void init() {
-        comments = new ArrayList<>();
-    }
-
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
 
-        Query query = new Query("Comment").addSort("date", SortDirection.DESCENDING);
+        String commentNumStr = getParameter(request, "number", "5");
+        int commentNum = 0;
```

Comment:
I think a better name for this variable would be something like `maxNumComments`. Something that shows that we will only display this number of comments if we don't run out of them first.